lib/chutney/TorNet.py

```diff
@@ -1412,7 +1412,7 @@ def combineDirInfoStatuses(self, dir_status, status_key_list,
                 continue
 
             (old_status_code, old_flav, old_msg) = dir_status
-            (new_status_code, new_flav, new_msg) = new_status
+            (new_status_code, new_flav, _) = new_status
```

Comment:
I don't know if this fix is useful :)

Reply:
It probably makes more sense to keep it there, for documentation purposes.